python/LAMBDA_INSIDE_VPC/parameters.json

```diff
@@ -0,0 +1,12 @@
+{
+  "Version": "1.0", 
+  "Parameters": {
+    "CodeKey": "LAMBDA_INSIDE_VPC.zip", 
+    "SourceRuntime": "python3.6", 
+    "RuleName": "LAMBDA_INSIDE_VPC", 
+    "SourceEvents": "AWS::Lambda::Function", 
+    "OptionalParameters": "{}", 
+    "InputParameters": "{\"subnetId\": \"subnet-124231, subnet-3532432\"}"
```

Comment:
Move this into the OptionalParameters. And let the value empty for pushing into this repo. The RDK will do the right thing and not include them. If it is in InputParameter, the RDK will retrun an error if empty.

Reply:
I tried giving below value for the `InputParameters` with the empty value for the subnetId and empty value to the InputParameters, 
```
    "InputParameters": "{\"subnetId\": \"\"}"
    "InputParameters": "{}"
```

In both cases RDK got deployed successfully and on the console it was showing Null as value for the SubnetId in the first and entire parameter as Null in second case. 

When i tried giving empty value on the console (GUI) it ignore if its empty and does not create any parameter at all. 

Looks like RDK is accepting the empty value, in that case should I keep the check or remove it ?

Reply:
Sorry, i was referring the comment 

`This cannot happen. Config will not pass an empty parameter.`